Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM tiangolo/uvicorn-gunicorn-starlette:python3.8
+
+# Install Poetry
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | POETRY_HOME=/opt/poetry python && \
+    cd /usr/local/bin && \
+    ln -s /opt/poetry/bin/poetry && \
+    poetry config virtualenvs.create false
+
+# Copy using poetry.lock* in case it doesn't exist yet
+COPY ./pyproject.toml ./poetry.lock* /app/
```

Comment:
As the current code,  we also want to copy `./config.toml` (if it exists) which contains the bot configuration.